server/functions.lua

```diff
@@ -209,6 +209,21 @@ function PaycheckLoop()
     SetTimeout(QBCore.Config.Money.PayCheckTimeOut * (60 * 1000), PaycheckLoop)
 end
 
+QBCore.Functions.SpawnVehicle = function(source, entity, locations, plate)
+    local CreateAutomobile = `CREATE_AUTOMOBILE`
+    local model = (type(entity) == "number" and entity or GetHashKey(entity))
```

Comment:
this is not a valid way of getting model hash from an entity, and if this is a spawn vehicle function - why are you using the entity handle as a parameter??